enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -2,21 +2,25 @@
 enterprise_learner_portal serializer
 """
 
+from opaque_keys.edx.keys import CourseKey
 from rest_framework import serializers
 
 from django.utils.translation import gettext as _
 
+from enterprise.constants import EXEC_ED_COURSE_TYPE, EXEC_ED_LANDING_PAGE
 from enterprise.utils import NotConnectedToOpenEdX
 from enterprise_learner_portal.utils import get_course_run_status
 
 try:
     from lms.djangoapps.bulk_email.api import get_emails_enabled
     from lms.djangoapps.certificates.api import get_certificate_for_user
     from lms.djangoapps.course_api.api import get_course_run_url
+    from openedx.core.djangoapps.catalog.utils import get_course_data
```

Comment:
A similar method already exists in edx-enterprise. Please see https://github.com/openedx/edx-enterprise/blob/a0efdc77f0d6f416a612976a161a9fc629e29d1d/enterprise/api_client/discovery.py#L261